### ISSUE TITLE
Mismatch results for inception_v2 in C_Api_Sample.cpp

### ISSUE THREAD

**Issue by tvlenin** [microsoft/onnxruntime#2709]
**Describe the bug**
We are trying inference images using the inception_v2 model provided in the ONNX model zoo, and we are getting wrong results, I searched about this issue and found this [test](https://github.com/microsoft/onnxruntime/blob/c06dbd83113f706d6c9769f3eb153a0eff737ead/onnxruntime/test/onnx/main.cc), where you specify that inception_v2 has a mismatch result:

> std::set<BrokenTest> broken_tests = {
> .
> .
> .
>       {"tf_inception_v2", "result mismatch"},
>  .
> .
> .
> };
> 

**System information**
- OS Platform and Distribution  : Linux Ubuntu 18.04
- ONNX Runtime installed from (source or binary): source
- ONNX Runtime version: v1.0.0
- Python version: 3.6.7
- GCC/Compiler version: v7.4.0

It is known why the model is failing, and there are also plans to repair this?
Is this an issue of onnx or onnxruntime?

Thanks

**Comment by snnn** [comment]
The inception_v2 model provided in the ONNX model zoo is not converted from tf. It isn't the same error as you found.

**Comment by snnn** [comment]
Is it possible for you to use a different model? For example, inception_v3 or v4, you can get instructions from https://github.com/microsoft/onnxruntime/tree/master/samples/c_cxx/imagenet and I've verified the accuracy is correct. 

The inception_v2 issue reported by you is most likely because the image preprocessing isn't right.  The model you used was converted from caffe2, and the author provided a set of test data there, onnxruntime passed the test.  So, onnxruntime itself should be correct.  You may contact the caffe2/pytorch team to get more info on this model. (e.g. how to do the preprocessing).

Another option is: you may get the inception_v2 model converted from [TF](https://www.tensorflow.org/). However, there is a tiny bug either in the converter side or in onnxruntime. We need help from the tensorflow converter team to get it fixed. (FYI @faxu)

**Comment by tvlenin** [comment]
Hi @snnn,  
Thank you for the information, 
We try with other models and it works fine to us.
Thanks
